src/index.js

```diff
@@ -1,7 +1,13 @@
 /* eslint-disable no-new-func, camelcase */
 /* globals __non_webpack__require__ */
 
-const realImport = new Function('modulePath', 'return import(modulePath)')
+const realImport = typeof process === "undefined" ? new Function('modulePath', 'return import(modulePath)') : async () => {
```

Comment:
This is a commonjs module and this can't be null.

Reply:
then the check for the process variable should be made in the runtime right?

Reply:
I don't understand the question. In Node.js, the process global is always available.

Reply:
But isn't this module also imported in the browser? I am sorry if I get this module wrong.

Reply:
This module is _not_ supposed to be used inside browsers.